include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -675,10 +678,16 @@ struct __parallel_radix_sort_iteration
     }
 }; // struct __parallel_radix_sort_iteration
 
+// sorting by just one work group
+#include "parallel_backend_sycl_radix_sort_one_wg.h"
+
```

Comment:
Earlier I temporarily changed the file to include into namespace, in order to simplify merges with other changes done recently. But now I think it does not make much sense to keep the implementation in a different file; if put right here, it will add about 100 lines to this file, which seems OK to me.

Reply:
agreed